dspace-server-webapp/src/main/java/org/dspace/app/rest/repository/ResourcePolicyRestRepository.java

```diff
@@ -52,9 +72,90 @@ public ResourcePolicyRest findOne(Context context, Integer id) {
         throw new RepositoryMethodNotImplementedException(ResourcePolicyRest.NAME, "findAll");
     }
 
-
     @Override
     public Class<ResourcePolicyRest> getDomainClass() {
         return ResourcePolicyRest.class;
     }
+
+    @PreAuthorize("hasAuthority('AUTHENTICATED')")
```

Comment:
due to https://jira.lyrasis.org/browse/DS-4199 this will be ineffective. In any case at this stage (in this PR) your are not yet testing the security constraints stated in the contract so it is fine for me